### PR TITLE
Review of Project from Admissions Trial 

### PR DESCRIPTION
**Pull request by felixodette**
Pushed a copy of the project from the admissions trial.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -73,22 +73,22 @@ <h4>MORE FROM YOUTUBE</h4>
     </div>
   </aside>
   <div class="categories">
```
Comment (felixodette) [inline index.html:75]: Please consider using semantic HTML tags.

File: index.html [hunk 2]
```diff
@@ -73,22 +73,22 @@ <h4>MORE FROM YOUTUBE</h4>
...
+      <button class="category" type="submit">Category 6</button>
+      <button class="category" type="submit">Category 7</button>
+      <button class="category" type="submit">Category 8</button>
+      <button class="category" type="submit">Category 9</button>
+    </div>
   </div>
   <div class="videos">
-    <section  class="video-section">
-      <article class="video-container">
+    <div  class="video-section">
```
Comment (felixodette) [inline index.html:90]: Consider using fewer containers in this section. Try to structure the elements in this section in a grid. 
* CSS Grid Layout excels at dividing a page into major regions or defining the relationship in terms of size, position, and layer, between parts of a control built from HTML primitives.

**Review by felixodette — COMMENTED** [review]
Great job! You're almost there!! 🎊 

However, there are a few changes to be made that will improve the quality of your code. Please incorporate the changes below ♻️ and resubmit your pull request once you're done.

**Review by aimemalaika — CHANGES_REQUESTED** [review]
Hi @felixodette, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

the-simpsons-homer-simpson

## Required Changes ♻️

- [ ] Kindly try to explain why you request the changes.
- For example:
   - This is a well-explained request: Why don't you use an article tag for the .first-article element instead of a div, that would improve the HTML structure and make the page more accessible
   - This request lacks a "why": Why don't you use an article tag here instead of a div?

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 3]
```diff
@@ -73,22 +73,22 @@ <h4>MORE FROM YOUTUBE</h4>
...
-      <button class="category">Category 2</button>
-      <button class="category">Category 3</button>
-      <button class="category">Category 4</button>
-      <button class="category">Category 5</button>
-      <button class="category">Category 6</button>
-      <button class="category">Category 7</button>
-      <button class="category">Category 8</button>
-      <button class="category">Category 9</button>
-    </section>
+    <div class="category-section">
```
Comment (felixodette) [inline index.html:76]: Please change `<div>` to `<section>`, thereby implementing semantic HTML.
* Semantic HTML refers to syntax that makes the HTML more comprehensible by better defining the different sections and layout of web pages. It makes web pages more informative and adaptable, allowing browsers and search engines to better interpret content.

**Review by felixodette — COMMENTED** [review]
Thanks @[aimemalaika](https://github.com/aimemalaika) for the quick review!

I've made the following changes:
* Explained why I feel the changes need to be made.

**Review by aimemalaika — APPROVED** [review]
Hi @felixodette,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

133591850-fdcb7dd3-dd7a-4274-81e7-0e59434148f7

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**